src/WebJobs.Script.WebHost/WebJobs.Script.WebHost.csproj

```diff
@@ -62,7 +62,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.4.1" />
+    <PackageReference Include="Azure.Identity" Version="1.9.0" />
```

Comment:
Is upgrading to the newer package enough? No other code changes needed on host side?